cmd/juju/commands/bootstrap.go

```diff
@@ -653,6 +651,15 @@ to create a new model to deploy k8s workloads.
 		}
 	}
 
+	cfgControllerName, _ := bootstrapCfg.controller[controller.ControllerName].(string)
```

Comment:
For now, let's not allow the controller name to be set via --config at all - we want to avoid 2 ways of doing the same thing.
So just check if bootstrapCfg.controller[controller.ControllerName] is not empty and complain if it is set